horizon/values.yaml

```diff
@@ -157,7 +157,10 @@ core:
   database:  # Access to the OpenNMS database
    username: opennms
    password: 0p3nNM5
-   poolSize: 50
+   idleTimeout: 3
+   loginTimeout: 600
+   minPool: 25
```

Comment:
Is this intentional to update default minPool to 25 ?

Reply:
It seems like this is driven from here: https://github.com/OpenNMS/opennms/blob/c908850620c436a6d568a173bb3fdb87161fdbda/opennms-base-assembly/src/main/filtered/etc/opennms-datasources.xml#L9